charts/opencost-config/templates/opencost/dashboards.yaml

```
@@ -1,3 +1,4 @@
+{{ if and .Values.dashboards }}
```

NIT: ```suggestion
{{ if .Values.dashboards }}
```
`and` would only be needed if there were multiple conditions, there is only one defined.